Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -70,6 +70,14 @@ public interface IGitModule
         /// <summary>Indicates HEAD is not pointing to a branch (i.e. it is detached).</summary>
         bool IsDetachedHead();
 
+        /// <summary>
+        /// Convert native (Windows) path to the path for the Git executable
+        /// (this could be for instance WSL)
+        /// </summary>
+        /// <param name="path">ge.exe path to repo.</param>
```

Comment:
What does this mean?

Reply:
To set inherit in GitModule
```suggestion
        /// <param name="path">Git Extensions native (Windows) path.</param>
```

Reply:
Refreshed the descriptions in general